conduct.md

```diff
@@ -213,3 +218,5 @@ Any decision made by the instructor for the given infraction(s) will be final.
 
 Janyl Jumadinova
 :heavy_exclamation_mark: TODO: Add your name
+Nate Angus
+Paige Downey
```

Comment:
I believe that there is a missing name that contributed to this PR but has not been added to the list